pkg/virt-operator/install-strategy/create.go

```diff
@@ -2261,6 +2335,12 @@ func SyncAll(queue workqueue.RateLimitingInterface, kv *v1.KubeVirt, prevStrateg
 		return false, err
 	}
 
+	// create/update config maps
+	err = createOrUpdateConfigMaps(kv, targetStrategy, stores, clientset, expectations)
```

Comment:
This may potentially override the `kubevirt-ca` configmap. I think you have to explicitly skip this configmap when comparing the  secrets.

Reply:
There is already check for that:
```
if cm.Name == components.KubeVirtCASecretName {
  continue
}
```
https://github.com/kubevirt/kubevirt/pull/3133/files#diff-d5561d66dec277d1e552dd8ac98e1cfcR680